opencog/scm/CMakeLists.txt

```diff
@@ -33,6 +33,18 @@ ADD_GUILE_MODULE (FILES
 	MODULE_DESTINATION "${GUILE_SITE_DIR}/opencog"
 )
 
+# Create three copies of (opencog as-config) module:
+SET(SCM_IN_BUILD_DIR "#t")
+# One for build dir
+CONFIGURE_FILE(opencog/as-config_TEMPLATE.scm ${GUILE_BIN_DIR}/opencog/as-config.scm)
```

Comment:
The industry-standard naming convention is to call this file `as-config.scm.in`  and not `as-config_TEMPLATE.scm` -- I prefer the industry-standard naming convention, as that makes it easier to understand at a glance.

Reply:
Makes sense - I've renamed it. Though, sadly, various editors/tools don't seem to detect it's a scheme file when `.in` is the extension.